prep-javascript-variables/index.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>JavaScript Variables</title>
+  </head>
+  <body>
+    <script>
+      var firstName = 'Chorong';
+      var lastName = 'Shin';
+      var fullName = firstName + lastName;
+      console.log('fullName:', fullName);
+
+      var numberOfPets = 0;
+      console.log('numberOfPets', numberOfPets);
+
+      var likesVegetables = 'true';
+      console.log('likesVegetables', likesVegetables);
```

Comment:
You should leave off the quotation marks from boolean values so that JavaScript doesn't interpret them as strings.